roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -114,12 +110,21 @@
     command: swapoff --all
 
   - name: Remove swap entries from /etc/fstab
+    replace:
+      dest: /etc/fstab
+      regexp: '(^[^#].*swap.*)'
+      replace: '# \1'
+      backup: yes
+
+  - name: Add notice about disabling swap
     lineinfile:
       dest: /etc/fstab
-      regexp: 'swap'
-      state: absent
+      line: '# OpenShift-Ansible Installer disabled swap per overcommit guidelines'
+      state: present
 
-  when: swap_result.stdout_lines | length > 0
+  when:
+  - swap_result.stdout_lines | length > 0
+  - openshift_disable_swap | default(true)
```

Comment:
The yamllint checks to make sure the _file_ is consistent.  So, if it starts out indented in the file, then it needs to be the same through the rest of the file.  When we started doing yamllint checks, we didn't specify that it _had_ to be one way or the other because there was quite a mix throughout the code base.
https://github.com/openshift/openshift-ansible/blob/master/.yamllint#L53